includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -33,15 +44,17 @@ class WC_Stripe_Apple_Pay_Registration {
 
 	public function __construct() {
 		add_action( 'init', array( $this, 'add_domain_association_rewrite_rule' ) );
+		add_action( 'admin_init', [ $this, 'verify_domain_on_domain_name_change' ] );
```

Comment:
Sorry to just comment on this now; Let's just the long array syntax `array()` instead of the short `[]` for uniformity; we're using the long syntax (seemingly) everywhere.